### PR TITLE
693 - landing page amends

### PR DESCRIPTION
**Pull request by omonk**
## Description - [Trello ticket](https://trello.com/c/BupfbsWx/693-post-launch-landing-page-amends)

### CODE REVIEW

File: src/components/Homepage/Intro.js [hunk 1]
```diff
@@ -37,6 +37,8 @@ const IntroRow = styled(Row)`
 `
 
 const StyledCardTitle = styled(CardTitle)`
+  font-weight: 400;
```
Comment (jeysal) [inline src/components/Homepage/Intro.js]: Looks like this is in `CardTitle` as `is('regular')`?

Comment (jeysal) [reply src/components/Homepage/Intro.js]: Kinda weird that an override is called "regular" :see_no_evil:

File: src/components/Homepage/OurWork.js [hunk 2]
```diff
@@ -96,7 +97,24 @@ const AnimatedLink = styled(Link)`
...
+
+  ${breakpoint('tablet')`
+    margin-top: ${({ theme }) => theme.space[5]};
+  `}
+`
+
+const CaseStudyImageWrapper = styled.div`
+  background-color: #${({ backgroundColor }) => backgroundColor};
+
+  > * {
```
Comment (jeysal) [inline src/components/Homepage/OurWork.js]: Why is this necessary?

File: src/components/Homepage/OurWork.js [hunk 3]
```diff
@@ -109,22 +127,20 @@ const CaseStudy = ({ caseStudy }) => {
...
-        width={100}
-        height={130}
-        color={posterColor}
-        image={alternativePreviewImage}
-      >
+      <Card width={100} height={130} color={posterColor}>
         <CardHeader backgroundColor={posterColor}>
           <section>
-            <BodyPrimary reverse={reverseColor} muted>
+            <BodyPrimary noPaddingTop reverse={reverseColor} muted>
```
Comment (jeysal) [inline src/components/Homepage/OurWork.js]: Using this as boolean attributes causes React warnings :/
Do we have any plans to improve the modifier handling to keep it easy to use but also avoid warnings

File: stories/__snapshots__/AboutUs.stories.snapshot [hunk 4]
```diff
@@ -1028,26 +1028,6 @@ exports[`Storyshots AboutUs Partners 1`] = `
 }
 
 .c6 {
-  -webkit-flex-wrap: wrap;
```
Comment (jeysal) [inline stories/__snapshots__/AboutUs.stories.snapshot]: Yeah, as we've talked about, I don't feel that most of the storyshots provide a lot of value in their current state :sweat_smile:

File: src/components/Homepage/OurWork.js (hunk 2 above)
Comment (omonk) [reply src/components/Homepage/OurWork.js]: Ah yes, should have put a comment in here. this is because at some sizes the gap between the image and the header section was big enough (1 pixel) to see the grey background of the section it's on. This just prevents that

File: src/components/Homepage/OurWork.js (hunk 3 above)
Comment (omonk) [reply src/components/Homepage/OurWork.js]: as per your slack comment we could probably refactor to remove the styled-is?

File: stories/__snapshots__/AboutUs.stories.snapshot (hunk 4 above)
Comment (omonk) [reply stories/__snapshots__/AboutUs.stories.snapshot]: ditto

File: src/components/Homepage/Intro.js (hunk 1 above)
Comment (omonk) [reply src/components/Homepage/Intro.js]: oh boy

Comment (jeysal) [reply src/components/Homepage/Intro.js]: So shall we use `regular` here instead of repeating the 400 constant?

File: src/components/Homepage/OurWork.js (hunk 2 above)
Comment (jeysal) [reply src/components/Homepage/OurWork.js]: Can we put it on the child itself?
